StartMenu.py

```diff
@@ -0,0 +1,355 @@
+# -*- coding: utf-8 -*-
+import pygame
+import configparser
+import json
+import re
+import os
+
+
+class ConfigHandler(object):
+    ''' Klasse behandelt alle Konfigs die geladen / gespeichert / veraendert werden
+    '''
+    Configurations = dict()
+
+    @staticmethod
+    def set(filename="standard", *args):
+        # setzt den jeweiligen Wert neu
+        config = ConfigHandler.Configurations[filename]    
+        if len(args) == 3:
+            value = json.loads(config.get(args[0], args[1]))
+            config.set(args[0], args[1], str(args[2]))
+            if value != args[2]:
+                ConfigHandler.save_file(filename)
+
+    @staticmethod
+    def load_dict(defaults, filename):
+        # Uebergebenes Dictionary muss immer zuerst eingelesen werden
+        # Konfig soll diese Standardwerte immer ueberschreiben
+        if filename not in ConfigHandler.Configurations:
+            ConfigHandler.Configurations[filename] = configparser.ConfigParser()
+            ConfigHandler.Configurations[filename].read_dict(defaults)
+            ConfigHandler.load_file(filename)
+    
+    @staticmethod
+    def save_file(filename):
+        if filename != "standard":
+            with open(filename, 'w') as configfile:
+                ConfigHandler.Configurations[filename].write(configfile)
+
+    @staticmethod
+    def load_file(filename):
+        # Datei einlesen mittels ConfigParser
+        if filename not in ConfigHandler.Configurations:
+            ConfigHandler.Configurations[filename] = configparser.ConfigParser()
+        if filename != "standard":
+            try:
+                with open(filename, 'r') as configfile:
+                    ConfigHandler.Configurations[filename].read_file(configfile)
+            except FileNotFoundError:
+                print("Datei existiert nicht, wenn Optionen geandert werden, wird diese geschrieben")
+
+    @staticmethod
+    def get_options(filename, sections):
+        ''' Example Usage
+            # (self.display, self.menu) = ConfigHandler.get_options(filename, ("display", "menu",))
+        '''
+        dict1 = dict()
+
+        ConfigHandler.load_file(filename)
+        config = ConfigHandler.Configurations[filename]
+        for section in sections:
+            dict1[section] = dict()
+            try: 
+                options = config.options(section)
+                for option in options:
+                    value = str(re.sub("'","\"",config.get(section, option)))
+                    try:
+                        dict1[section][option] = json.loads(value)
+                    except ValueError:
+                        # Fehler tritt auf, weil in value kein valider Syntax fuer json ist
+                        dict1[section][option] = config.get(section, option)
+
+            except configparser.NoSectionError:
+                print("Auswahl nicht vorhanden: %s" % section)
+        
+        # falls nur ein Abschnitt geholt werden muss
```

Comment:
Wenn ich die get_options(.,sections) mit einem 2-Werte Tuple aufrufe returne ich ein auch ein 2-Werte Tuple. Das wird dann anschließend auch in 2 Variablen entpackt.  
Bsp.: self.display, self.colors = ConfigHandler.get_options(self.filename, ("display", "colors"))

Ich kann aber schlecht ein Tuple in nur eine Variable entpacken. Deswegen habe ich die Prüfung eingebaut.
Bsp.: self.display = ConfigHandler.get_options(self.filename, ("display",))

Scheint so aber schlecht gelöst zu sein!?
Dachte, so wäre es besser nutzbar.